src/protocol/mod.rs

```diff
@@ -203,6 +210,10 @@ impl From<u8> for CommandType {
             0x02 => CommandType::DeviceCapabilities,
             0x03 => CommandType::DeviceId,
             0x04 => CommandType::DeviceInfo,
+            0x81 => CommandType::GetDigests,
+            0x82 => CommandType::GetCert,
+            0x83 => CommandType::Challenge,
```

Comment:
Whoops.

Mind putting this in its own commit to the effect of "[protocol] Add missed entries to `ComandType::from()`"?

`git add -p` is your friend here. =)

Reply:
Done